Makefile

```diff
@@ -19,13 +19,15 @@
 
 # values used in workspace-status.sh
 
-DOCKER_REGISTRY?=us.gcr.io/chris-love-operator-playground
+DOCKER_REGISTRY?=quay.io/alinalion
```

Comment:
Can we not set this please.  We do not want private repos in our Makefile

Reply:
ok